internal/kafka/command_share_group_consumer_list_test.go

```diff
@@ -0,0 +1,116 @@
+package kafka
```

Comment:
We generally don't write unit tests for CRUD command implementations since the integration tests should be able to provide the same coverage and are easier to maintain.

The implementations of these commands look pretty standard without anything unusual that would benefit from a unit test, so I think we can probably remove most or all of these.